apollo-router/src/plugins/telemetry/tracing/apollo_telemetry.rs

```diff
@@ -135,19 +137,25 @@ pub(crate) enum Error {
     SystemTime(#[from] SystemTimeError),
 }
 
+// TBD(tim): maybe we should move this?
+// Also, maybe we can just use the actual SpanData instead of the light one?
```

Comment:
The main idea of having this LightSpanData is to consume less memory in the LRU cache. it was a good improvement when introduced

Reply:
Yeah another approach would be grouping by trace and attributes to get rid of the duplications, which would be nice for the wire protocol as well.  Probably something to visit in the next phase when we feel good about moving entirely to OTLP.

Reply:
Seems like after passing the first round of perf tests we can leave this as-is.